data/json/items/comestibles/egg.json

```diff
@@ -742,21 +742,22 @@
   {
     "type": "COMESTIBLE",
     "id": "rehydrated_eggs",
-    "copy-from": "egg_bird",
-    "name": { "str": "rehydrated egg" },
+    "name": { "str_sp": "liquid eggs" },
     "weight": "50 g",
+    "spoils_in": "4 days",
+    "comestible_type": "FOOD",
     "color": "yellow",
-    "symbol": "%",
-    "looks_like": "powder_egg",
-    "quench": 2,
+    "symbol": "~",
+    "quench": 4,
     "calories": 80,
-    "description": "Rehydrated eggs.  You can use them instead of eggs when cooking, provided you don't care that they have lost their complete shape.",
-    "price": 22,
-    "price_postapoc": 25,
+    "description": "Liquid eggs that contains almost all the nutrients of eggs.  If you accidentally drop the eggshell fragments into it, it may be able to add a little extra calcium.",
```

Comment:
```suggestion
    "description": "Rehydrated eggs.  You can use them instead of eggs when cooking, provided you don't care that they have lost their complete shape.",
```